ocs_ci/deployment/aws.py

```diff
@@ -61,6 +61,17 @@ def __init__(self):
         # check or during a cluster cleanup)
         self.cluster_prefixes_special_rules = CLUSTER_PREFIXES_SPECIAL_RULES
 
+    def deploy_ocp(self, log_cli_level="DEBUG"):
+        super(AWSBase, self).deploy_ocp(log_cli_level)
+        ocp_version = version.get_semantic_ocp_version_from_config()
+        ocs_version = version.get_semantic_ocs_version_from_config()
+
+        if ocs_version >= version.VERSION_4_10 and ocp_version >= version.VERSION_4_9:
+            # The default storage class is gp3-csi for OCS 4.10 and above
+            self.DEFAULT_STORAGECLASS = config.DEPLOYMENT.get(
```

Comment:
please don't change the default in ocs-ci for AWS storageclass. We first need to do the evaluation with gp3-sci and then decide along with PM if that's the way to go

Reply:
in case it will be decided to change the default to gp3-csi, we will need to do a wide communication to the QE and dev teams.
Apart from that, changing the default is not done here but assigning the storageclass name to DEFAULT_STORAGECLASS in ocs_ci/deployment/aws.py:52

Reply:
Will change the default SC to be `gp2`. About the second comment, we had better change the default SC here, because we don't want to change the deployments with OCS 4.9 and below. Just from ODF 4.10 and above with use the new SCs.